docs/current_release/download.md

```diff
@@ -54,8 +54,8 @@ Development sources
 Installers and images
 ---------------------
 
-| **Mac OS packages**        |  dealii-9.6.0-sonoma-arm64.dmg, see [github](https://github.com/dealii/dealii/releases/download/v9.6.0/dealii-9.6.0-sonoma-arm64-clang15.dmg)  <br>  (PGP signature file: [github](https://github.com/dealii/dealii/releases/download/v9.6.0/dealii-9.6.0-sonoma-arm64-clang15.dmg.asc)) <br> August 31, 2024, 1.71GB.  <br>  [Mac OSX Instructions](https://github.com/dealii/dealii/wiki/MacOSX) |
------------: |:-------------|          
+| **Mac OS packages**        |  dealii-9.6.2-sequoia-arm64-clang16.dmg, see [github](https://github.com/dealii/dealii/releases/download/v9.6.2/dealii-9.6.2-sequoia-arm64-clang16.dmg)  <br>  (PGP signature file: [github](https://github.com/dealii/dealii/releases/download/v9.6.2/dealii-9.6.2-sequioia-arm64-clang16.dmg.asc)) <br> February 5, 2025, 983MB.  <br>  [Mac OSX Instructions](https://github.com/dealii/dealii/wiki/MacOSX) |
```

Comment:
Looks like the PGP file has a typo, `sequioia` instead of `sequoia`, not that it matters that much.
How did we reduce the size by ~50%?

Reply:
Well spotted!

The size is reduced by 50% by simply not including vtk. There was no way I could compile vtk without errors, and at some point I just gave up.